### DISCUSSION TITLE
Pagedown/Flexdown in Quarto

### DISCUSSION THREAD

**Discussion by bsgallih12** [quarto-dev/quarto-cli#4135] [Q&A] [answered]
Hi,

This summer Tom Mock gave a talk about Quarto and suggested that the Quarto team was working on a Fall 2022 timeline to bring pagedown to Quarto. Has that happened yet either at the stable or developmental level? Further, has flexdown been ported over yet?

**Comment by mcanouil** [comment, marked as answer]
If you don't see it in the releases changelog, this means it is not ported/implemented.
Furthermore, if Quarto team has something to announce they will.

See related discussions/issues: #705, #2205, #640, #3271, #843 (I probably missed some).

To summarise, be patient, and read releases and pre-releases notes to see what's new and coming: <https://quarto.org/docs/download/>.

**Reply by bsgallih12** [reply]
Thank you!
